src/hooks/__snapshots__/Trades.test.ts.snap

```diff
@@ -0,0 +1,2000 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Trade should generate proper route pairs for token pair with one route-unrestricted and one route-restricted token 1`] = `
+Array [
+  Array [
+    Token {
+      "address": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+      "chainId": 1,
+      "decimals": 6,
+      "name": "USD//C",
+      "symbol": "USDC",
+    },
+    Token {
+      "address": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+      "chainId": 1,
+      "decimals": 18,
+      "name": "Wrapped Ether",
+      "symbol": "WETH",
+    },
+  ],
+  Array [
+    Token {
+      "address": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+      "chainId": 1,
+      "decimals": 6,
+      "name": "USD//C",
+      "symbol": "USDC",
+    },
+    Token {
+      "address": "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+      "chainId": 1,
+      "decimals": 18,
+      "name": "Dai Stablecoin",
+      "symbol": "DAI",
+    },
+  ],
+  Array [
+    Token {
+      "address": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+      "chainId": 1,
+      "decimals": 6,
+      "name": "USD//C",
+      "symbol": "USDC",
+    },
+    Token {
+      "address": "0xdAC17F958D2ee523a2206206994597C13D831ec7",
+      "chainId": 1,
+      "decimals": 6,
+      "name": "Tether USD",
+      "symbol": "USDT",
+    },
+  ],
+  Array [
+    Token {
+      "address": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+      "chainId": 1,
+      "decimals": 6,
+      "name": "USD//C",
+      "symbol": "USDC",
+    },
+    Token {
+      "address": "0xc00e94Cb662C3520282E6f5717214004A7f26888",
+      "chainId": 1,
+      "decimals": 18,
+      "name": "Compound",
+      "symbol": "COMP",
+    },
+  ],
+  Array [
+    Token {
+      "address": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+      "chainId": 1,
+      "decimals": 6,
+      "name": "USD//C",
+      "symbol": "USDC",
+    },
+    Token {
+      "address": "0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2",
+      "chainId": 1,
+      "decimals": 18,
+      "name": "Maker",
+      "symbol": "MKR",
+    },
+  ],
+  Array [
+    Token {
+      "address": "0xD46bA6D942050d489DBd938a2C909A5d5039A161",
+      "chainId": 1,
+      "decimals": 9,
+      "name": "Ampleforth",
+      "symbol": "AMPL",
+    },
+    Token {
+      "address": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+      "chainId": 1,
+      "decimals": 18,
+      "name": "Wrapped Ether",
+      "symbol": "WETH",
+    },
+  ],
+  Array [
+    Token {
+      "address": "0xD46bA6D942050d489DBd938a2C909A5d5039A161",
+      "chainId": 1,
+      "decimals": 9,
+      "name": "Ampleforth",
+      "symbol": "AMPL",
```

Comment:
AMPL DAI is a bug.

Reply:
My bad, I thought it was USDC/AMPL, this is right
https://www.ampltalk.org/app/forum/technology-development-17/topic/supported-dex-pools-61/